modules/ROOT/pages/create-community.adoc

```diff
@@ -105,6 +105,8 @@ After you create a community, you must create different user profiles for your c
 
 Before configuring member and guest profiles and permissions, open *Setup* > *User Management Settings* and ensure that *Enhanced Profile User Interface* is enabled.
 
+Use the following steps to configure custom user settings. To configure users settings automatically with default settings, use the *FIX* buttons in the xref:diagnostics.adoc#user-access-tab[API Community Manager diagnostics user access tab].
```

Comment:
Per the style ref, to ensure topic independence, we try to avoid using "the following" or similar overt references to content outside of the current topic, even if that topic is a container.

But more important...why do we need "the following" subtopic? There is really not enough (green) content here to require it, and a solitary subtopic is a red flag, pointing to a possible organizational issue. The peer topics don't have subs; why not make this topic parallel to them?

Noting this made me look further into this topic, including the hidden, legacy text. It looks like this content could use an overall organizational edit. For example, the following Configure topic is way, way too long and nested, and includes far too much reference material, which should be pulled out into a separate topic. Consider creating a JIRA to have an organizational edit done on this deliverable so that we might work on this together.

Here is my suggestion in the meantime:

```suggestion
Configure Profiles and Permissions
After you create a community, you must create different user profiles for your community _members_ (signed in) and _guests_, so that you can control access to and visibility of APIs and content. You can create these profiles automatically, using the *FIX* buttons in the xref:diagnostics.adoc#user-access-tab[API Community Manager diagnostics user access tab], or you can configure them manually.

[Include some statement about why a user would choose one method over the other.
Then follow this intro with separate topic sets for member users vs. guest users, including the Prerequisite or first step of ensuring that *Enhanced Profile User Interface* is enabled.]
```

Reply:
Updated with this suggestion. Also created a Jira for further reorganization in https://www.mulesoft.org/jira/browse/DOCS-11514 .